students/guillaume/quarter_2_project/leankitbackend/leankitbackend/leankitcards.py

```diff
@@ -0,0 +1,297 @@
+from collections import defaultdict
+import types
+from datetime import datetime
+
+'''
+Maybe define a Project class
+'''
+
+lanes_current_sprint = ['Current Sprint Backlog', 'Doing Now', 'Under Review',
+                        'Finished As Planned']
+
+lanes_Backlogs_next_sprint = ['Backlog', 'Next Sprint Planning']
+
+
+# Attention sub columns of recently finished / ready to archived has the
+# same name
+lanes_Finished = ['Finished As Planned', 'Started but not Finished',
+                  'Discarded or Obsolete']
+
+
+class Leankit_Cards:
+    def __init__(self, cards_list=None, date=None):
+        self.date = date
+        if cards_list is not None:
+            self.append(cards_list, date)
+
+    def append(self, cards_list, date=None):
+        if isinstance(cards_list, list):
```

Comment:
what else might it be? and if it is something else, what should happen?

I think you can simply assume it's a sequence, and loop through it.

maybe catch the exception if that doesn't work.